core-processor/src/common.rs

```diff
@@ -498,6 +498,9 @@ pub enum SystemExecutionError {
     /// Error during `into_ext_info()` call
     #[display(fmt = "`into_ext_info()` error: {_0}")]
     IntoExtInfo(MemoryError),
+    /// Incoming dispatch store has too many outgoing messages total bytes.
+    #[display(fmt = "Incoming dispatch store has too many outgoing messages total bytes")]
+    MessageStoreOutgoingBytesOverflow,
```

Comment:
This error will be returned from `core_processor::process`

Reply:
**IMPORTANT**: By the way, this error can happen if some already existed message exceed outgoing bytes limit, so this will cause panic in runtime... So may be we must process this case as message execution error.

Reply:
I've change this temporary: instead of returning `SystemExecutionError::MessageStoreOutgoingBytesOverflow`, we return now `ActorExecutionError::UnsupportedMessage`. After next runtime upgrade we should return back to system error.

Reply:
Yes

Reply:
commented